tools/integration_tests/util/setup/setup.go

```diff
@@ -164,6 +172,17 @@ func SetUpTestDir() error {
 	return nil
 }
 
+// Removing bin file after testing.
+func RemoveBinFileCopiedForTesting() {
```

Comment:
can we not call this method from unmount or executeTest below?

Reply:
No, because we are [building](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/operations/operations_test.go#L104) gcsfuse for every test package once only, unmount getting called after running test for every flag set. 
e.g.
[here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/operations/operations_test.go#L88) four times unmount getting called.
executeTest is getting called to run  test for flags for different mountings.
e.g [here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/operations/operations_test.go#L106) for static mounting, only_dir_mounting,... - execute tests getting called
here